numpy/core/code_generators/generate_umath.py

```diff
@@ -904,7 +907,14 @@ def english_upper(s):
           "PyUFunc_SimpleBinaryOperationTypeResolver",
           TD(ints),
           TD('O', f='npy_ObjectLCM'),
-          )
+          ),
+'matmul' :
+    Ufunc(2, 1, None,
+          'XXX TODO: get docstring',
```

Comment:
Isn't setting in `ufunc_generators` meant to be the final state? Maybe adjust this to something like `Placeholder: set in ufunc_generators.py` (and if it is not meant to be the final state, then add an actual comment).

Reply:
Added the docstring from `numpy.core.umath.matmul` like other ufuncs